src/CMakeLists.txt

```diff
@@ -7,15 +7,10 @@ set(CMAKE_COMPILE_WARNING_AS_ERROR ON)
 if(MSVC)
     add_subdirectory(dll)
 else()
-    #file(GLOB CPP_SOURCES *.cpp)
-    set(CPP_SOURCES abstract_data_store.cpp ann_exception.cpp disk_utils.cpp 
-        distance.cpp index.cpp in_mem_graph_store.cpp in_mem_data_store.cpp
-        linux_aligned_file_reader.cpp math_utils.cpp natural_number_map.cpp
-        in_mem_data_store.cpp in_mem_graph_store.cpp
-        natural_number_set.cpp memory_mapper.cpp partition.cpp pq.cpp
-        pq_flash_index.cpp scratch.cpp logger.cpp utils.cpp filter_utils.cpp)
+    file(GLOB CPP_SOURCES *.cpp)
```

Comment:
please revert to listing the files. This is not recommended: https://cmake.org/cmake/help/latest/command/file.html?highlight=glob#glob

Reply:
changed.